packages/sw-runtime-caching/src/lib/stale-while-revalidate.js

```diff
@@ -24,7 +24,10 @@ import assert from '../../../../lib/assert';
  *
  * In addition to updating the appropriate caches, it will also trigger any
  * appropriate plugins defined in the underlying `RequestWrapper`.
- * XXXXXXXX
+ * 
+ * This class always returns the cache and returns it first regardless of how
+ * quickly a network response returns. This make it suitable for scenarios where
+ * a fastest strategy would otherwise be used.
```

Comment:
The descriptions don't match. The `sw-toolbox`'s `fastest` was [Cache & network race](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-and-network-race), and now we have an actual [Stale-while-revalidate](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#stale-while-revalidate).

I think the idea was that we were going to tell people who previously used `fastest` in `sw-toolbox` that `StaleWhileRevalidate` was the closest equivalent, despite not being exactly the same.

Using the phrase "fastest" here without any qualification doesn't make sense, since "fastest" doesn't have any generic meaning when it comes to cache strategies. "fastest" only make sense within the context of `sw-toolbox`, so the second sentence could be reworked to specifically link to the [`sw-toolbox.fastest` documentation](https://googlechrome.github.io/sw-toolbox/api.html#toolboxfastest), and make it clear that it's information that's useful for people migrating from `sw-toolbox`.